### ISSUE TITLE
Shortcuts for quick bookmark "7" and quick bookmark "0" conflicts with other functions

### ISSUE THREAD

**Issue by PeterMortensen** [Komodo/KomodoEdit#3857]
### Short Summary

Two of the keyboard shortcuts for quick bookmarks, <kbd>Ctrl</kbd> + <kbd>Shift</kbd>  + <kbd>7</kbd> and <kbd>Ctrl</kbd> + <kbd>Shift</kbd>  + <kbd>0</kbd> does not set the expected bookmark. Instead, some line outcommenting and zooming happen, respectively.

This was tried in a file with file extension ".js" and can also be reproduced when trimmed down to a single line.

### Steps to Reproduce

1. Make a new file
2. Type "ASD"
3. Save it as someFile.js
4. Press <kbd>Ctrl</kbd> + <kbd>Shift</kbd>  + <kbd>7</kbd>
5. "//" will be prepended to "ASD", instead of setting the bookmark
6. The same <kbd>Ctrl</kbd> + <kbd>Shift</kbd>  + <kbd>0</kbd>. It will
 zoom instead of setting the bookmark

### Expected results

Bookmark 7 and Bookmark 0 set, respectively

### Actual results

"//" is be prepended to "ASD" and the document is zoomed into, respectively.

### Platform Information

"Komodo IDE, version 12.0.1, build 91869, platform linux-x86_64.
Built on Mon Feb 10 19:37:24 2020."

Ubuntu 19.10, but using Cinnamon instead of GNOME (with its broken clipboard).

**Comment by th3coop** [comment]
Have you confirmed that they haven't been changed in your keybinding prefs?

**Comment by th3coop** [comment]
*Edit menu > Preferences > Keybindings*

**Comment by PeterMortensen** [comment]
It was the out-of-the-box experience, right after installation. I haven't changed any configuration (except perhaps answering startup/one-time questions). "Keybinding scheme" is set to "Default".

Do you mean you couldn't reproduce the problem?

**Comment by PeterMortensen** [comment]
I don't know if it is relevant, but in the terminal window at startup there is output:

    (process:20291): GLib-CRITICAL **: 02:42:49.411: g_slice_set_config: assertion 'sys_page_size == 0' failed

**Comment by th3coop** [comment]
No, i can't reproduce the issue.  I don't think the GLib output or memory availability is relevant here.  It the former were an issue Komodo would likely not start.  The latter would just result in a slow Komodo, not keybindings misfiring.

Can you double check your keybindings in your prefs?  The actions your described are not the default keybindings.  Maybe they changed somehow but I'd like to narrow things down.  I'm guessing that if you didn't change them in komodo then something outside Komodo is usurping the keybinding which is not at all uncommon, though usurping to comment out code would be really werid.
